src/index.js

```diff
@@ -309,6 +309,17 @@ const indy = {
     throw new Error(`Unsupported operation! Platform: ${Platform.OS}`)
   },
 
+  async listMyDidsWithMeta(wh) {
+    return IndySdk.listMyDidsWithMeta(wh)
```

Comment:
It seems like the iOS version is not implemented yet. Could you add the following error to indicate this:

```js
if (Platform.OS === 'ios') {
  throw new Error(`Unsupported operation! Platform: ${Platform.OS}`)
}
```

Reply:
seems like it's implemented 

https://github.com/hyperledger/indy-sdk/blob/7f4fbb055478181761347a981544fb823d6f47ba/wrappers/ios/libindy-pod/Indy/Wrapper/IndyDid.h#L218-L229

but I'm not 100% sure an don't have a good way to test IOS at the moment. Thanks

Reply:
I mean in the iOS part of this library. See https://github.com/absaoss/rn-indy-sdk/tree/master/ios/IndySdk.swift

Or am I missing something?

Reply:
whoops you're 100% right. I'll update the code to throw an error. Thanks for the feedback! 🙏